src/components/IconButton.jsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import { MdOutlineRemoveCircle } from 'react-icons/md';
+
+export default function IconButton({ variant, onClick }) {
```

Comment:
NON-BLOCKING: If you're going the route of variants, you could just create one button component instead of multiple. Just a suggestion and may not work for your long term plans!